scripts/utils.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function writeEula {
+  if ! echo "# Generated via Docker
+# $(date)
+eula=${EULA,,}
```

Comment:
What does `,,` do in `eula=${EULA,,}`

Reply:
converts the value of  $EULA to lowercase